docs/orchestration/concepts/api_keys.md

```diff
@@ -199,29 +199,54 @@ mutation {
 
 ## Using API keys with older versions of Prefect
 
-The `prefect auth login` command will not work with API keys and the  `PREFECT__CLOUD__API_KEY` setting will be ignored before version 0.15.0. In older versions, there were authentication tokens. Keys can be used in-place in older versions by setting them in the config or the environment in the `PREFECT__CLOUD__AUTH_TOKEN` setting.
+::: warning
+API tokens are no longer supported as an authentication method.
 
-Using an API key as a token for registering flows
+If you are running a version of Prefect older than 0.15.0, this section describes how you can use API keys for authentication.
+
+If you have logged in with an API key, but a token still exists on your machine, the API key will be used and the token will be ignored.
+:::
+
+If you are running a version of Prefect older than 0.15.0, note that:
+
+- The `prefect auth login` CLI command will not work with API keys.
+- The `PREFECT__CLOUD__API_KEY` setting will be ignored. 
+
+Versions prior to Prefect 0.15.0 used _authentication tokens_. You can use API keys in these older versions of Prefect by setting them in the config or the environment in the `PREFECT__CLOUD__AUTH_TOKEN` setting.
+
+Using an API key as a token for registering flows:
 ```bash
-export PREFECT__CLOUD__AUTH_TOKEN="<YOUR-KEY>"
+$ export PREFECT__CLOUD__AUTH_TOKEN="<YOUR-KEY>"
 ```
 
-Using an API key as a token for starting an agent by CLI
+Using an API key as a token for starting an agent by CLI:
 ```bash
 $ prefect agent local start -k "<SERVICE_ACCOUNT_API_KEY>"
 ```
 
-Using an API key as a token for starting an agent by environment
+Using an API key as a token for starting an agent by environment:
 ```bash
 $ export PREFECT__CLOUD__AGENT__AUTH_TOKEN="<YOUR-KEY>"
 $ prefect agent local start
 ```
 
+`PREFECT__CLOUD__AUTH_TOKEN` is no longer passed through to non-container flow run jobs.
+
+`PREFECT__CLOUD__AUTH_TOKEN` is set to the value of `PREFECT__CLOUD__API_KEY` for containerized flow run jobs, which will allow containers running old versions of Prefect to work with current agents with the caveat that the API key must be used with the default tenant.
```

Comment:
This is support for when:
- You are running an agent with 0.15.0+
- You are using an API key to authenticate your agent
- Your flow run is in a container with version <0.15.0

Perhaps this should be prefaced with a "As of version 1.0.0 ..."?

This section is for if you are using Prefect <0.15.0 but still want to use an API key instead of a token. I'm not sure this belongs here.

Reply:
I'm taking this out for now.